### PR TITLE
Revert to shell script for build version

### PR DESCRIPTION
**Pull request by neelimamukiri**
Also chmod the log files we create to allow user to deleted them.

Fixes https://github.com/contiv/ccn/issues/258 and removes the python requests dependency for getting build version (fixes https://github.com/contiv/install/issues/85). Allows user to override BUILD_VERSION as well.

### CODE REVIEW

File: install/ansible/uninstall_swarm.sh [hunk 1]
```diff
@@ -142,4 +143,4 @@ ansible_mount="-v $(pwd)/ansible:/ansible:Z"
 config_mount="-v $src_conf_path:$container_conf_path:Z"
 cache_mount="-v $(pwd)/contiv_cache:/var/contiv_cache:Z"
 mounts="$install_mount $ansible_mount $cache_mount $config_mount"
-docker run --rm $mounts $image_name sh -c "./install/ansible/uninstall.sh $netmaster_param -a \"$ans_opts\" $uninstall_scheduler -m $contiv_network_mode -d $fwd_mode $aci_param $reset_params $cluster_param"
+docker run --rm --net=host $mounts $image_name sh -c "./install/ansible/uninstall.sh $netmaster_param -a \"$ans_opts\" $uninstall_scheduler -m $contiv_network_mode -d $fwd_mode $aci_param $reset_params $cluster_param"
```
Comment (dseevr) [inline install/ansible/uninstall_swarm.sh]: Why did this become `--net=host`?

File: scripts/get_latest_release.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+# Try to get the latest release
+releases=$(curl -s https://api.github.com/repos/contiv/install/releases/latest)
+if [[ "$releases" != *"browser_download_url"* ]]; then
+	releases=$(curl -s https://api.github.com/repos/contiv/install/releases)
+	if [[ "$releases" != *"browser_download_url"* ]]; then
+		exit 1
+	fi
+fi
+release=$(echo "$releases" | python -c 'import json, sys;print json.load(sys.stdin)[0]["name"]')
```
Comment (dseevr) [inline scripts/get_latest_release.sh]: Nice 👍

File: install/ansible/install.sh [hunk 3]
```diff
@@ -92,16 +92,16 @@ env_file=install/ansible/env.json
 # Verify ansible can reach all hosts
 
 echo "Verifying ansible reachability"
-ansible all $ans_opts -i $host_inventory -m setup -a 'filter=ansible_distribution*' >& $inventory_log
-egrep 'FAIL|UNREACHABLE' $inventory_log >& /dev/null
+ansible all $ans_opts -i $host_inventory -m setup -a 'filter=ansible_distribution*' >&$inventory_log
```
Comment (dseevr) [inline install/ansible/install.sh]: Good to know `>&` passes shfmt... TIL

**Comment by dseevr** [comment]
Just a few comments, LGTM otherwise

File: install/ansible/uninstall_swarm.sh (hunk 1 above)
Comment (neelimamukiri) [inline install/ansible/uninstall_swarm.sh]: I made this change to the installer, but missed it in the uninstaller. Hit the issue on Linux while testing the access privileges for uninstall log file and added this.

Comment (neelimamukiri) [inline install/ansible/uninstall_swarm.sh]: No, it won't. That is a vagrant + VPN issue.

**Pull request merged**